WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java

```diff
@@ -104,9 +103,7 @@
     private static final int SELECT_LIBRARY_MENU_POSITION = 100;
     private static final int CLEAR_FEATURED_IMAGE_MENU_POSITION = 101;
 
-    private PostModel mPost;
-    private SiteModel mSite;
-
+    private PostSettingsListener mListener;
```

Comment:
Hmm yeah it's a tough one. It's a thing that provides you with a post model, so perhaps a `PostModelProvider`? Or something more descriptive, like `EditPostActivityHook`?

By the way, does this need to provide a `SiteModel`? We can't change the site from `EditPostActivity`, so passing it through as before doesn't seem like it would cause any problems.